### PR TITLE
TFS171190 add longest scan vector length

### PR DESCRIPTION
**Pull request by neverhood311**
I'm not sure why there were so many unrelated changes when I generated the CSharp client. Anyone know why?

### CODE REVIEW

File: src/SimulationCSharpClient/Client/SimulationClientGenerated.cs [hunk 1]
```diff
@@ -17115,6 +17131,10 @@ public partial class PartBasedSimulationParameters : System.ComponentModel.INoti
         private System.Collections.ObjectModel.ObservableCollection<double> _distortionAfterCutoffScaleFactorValues;
         private PartBasedSimulationParametersSupportType _supportType;
         private int? _voxelSampleRate;
+        private bool? _includeOnPlateStrainOutput;
+        private bool? _enableContours;
+        private PartBasedSimulationParametersLoadSteppingType? _loadSteppingType;
+        private int? _loadSteps;
```
Comment (neverhood311) [inline src/SimulationCSharpClient/Client/SimulationClientGenerated.cs:17137]: I'm not entirely sure why this change is here. This has nothing to do with my changes to the simulation api spec

File: version.properties [hunk 2]
```diff
@@ -1 +1 @@
-VERSION_TAG=3.40.1
+VERSION_TAG=3.41.0
```
Comment (neverhood311) [inline version.properties]: Does this version need to strictly match the version of the Simulation API Specification it's targeting? If so, I need to change this to 3.44.0.

**Review by chadqueen — APPROVED** [review]
